.travis.yml

```diff
@@ -28,17 +33,25 @@ compiler:
 env:
   matrix:
   - Q_OR_C_MAKE=cmake
+    QT_VERSION="59" # actually 5.9
   - Q_OR_C_MAKE=qmake
+    QT_VERSION="59" # actually 5.9
   global:
   - secure: VFI3UCiDrp47WTcUhsatdQvvWg+3gk00eBMZgSOXXKY5+hk+NOX7bOFcIM5t9nlZDbpHDr10SFTuUOw+PeWmLpFO06Zrjg86M9jm9WS4i8Cs9hfxoT6H4isXlR1vubX2LmNlHyzg8WtdNanlsufgecyaGksJxr7tVhG/cWyD6yo=
 matrix:
   exclude:
   - os: osx
     compiler: gcc
+  include:
+  - os: linux
+    compiler: gcc
+    env: 
+    - Q_OR_C_MAKE="cmake"
```

Comment:
Yeah, I can leave them out, they are pasted into the shell as they are there, which in turn strips the quotes out. But for consistency I'll remove them (once after I can check everything else works as it should, but travis is having hickups again)